### PR TITLE
[FIX] Standardise use of "entity" definition across specification

### PR DESCRIPTION
**Pull request by Lestropie**
Relates to discussion in bids-standard/bids-specification#947.

Consistently using the term "entity" to refer to key-value pairs within filenames provides a disambiguation from key-values that are stored in sidecar metadata files.

While I can definitely see a potential benefit in providing hyperlinks to the "Common principles - Definitions" entries whenever these defined terms are used in the spec, that's likely a much larger changeset, so may be deferred to a separate Issue at bids-standard/bids-specification if there is interest.

### CODE REVIEW

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 1]
```diff
@@ -217,8 +217,8 @@ If the structural images included in the dataset were defaced (to protect
 identity of participants) one MAY provide the binary mask that was used to
 remove facial features in the form of `_defacemask` files.
 In such cases,  the OPTIONAL [`mod-<label>`](../99-appendices/09-entities.md#mod)
-key/value pair corresponds to modality suffix,
-such as T1w or inplaneT1, referenced by the defacemask image.
+entity corresponds to modality suffix,
+such as `T1w` or `inplaneT1`, referenced by the defacemask image.
```
Comment (Lestropie) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:221]: Note: unrelated formatting change

**Comment by Lestropie** [comment]
There were uses of "key-value-pairs", "key-value", "key:value", "key/value", "keyword-value", and "parameter" all being used to refer to entities.

**Review by sappelhoff — COMMENTED** [review]
this is great, thanks a lot @Lestropie - would you mind opening a PR to bids-specification with this?

> While I can definitely see a potential benefit in providing hyperlinks to the "Common principles - Definitions" entries whenever these defined terms are used in the spec, that's likely a much larger changeset, so may be deferred to a separate Issue at bids-standard/bids-specification if there is interest.

agreed!

**Comment by Lestropie** [comment]
> would you mind opening a PR to bids-specification with this?

Plan was to merge this first, and it would then be integrated into bids-standard/bids-specification#947, as this PR targets the base branch of that PR. I can't currently propose this change as a standalone PR on bids-specification, as there is no suitable base branch on the upstream repo for construction of such.

Are you happy for this to be integrated into bids-standard/bids-specification#947, or would you prefer to go through the gymnastics of having essentially this same proposal as a standalone over on bids-standard/bids-specification?

**Comment by sappelhoff** [comment]
> Are you happy for this to be integrated into bids-standard#947, or would you prefer to go through the gymnastics of having essentially this same proposal as a standalone over on bids-standard/bids-specification?

no, that seems fine to me. No additional gymnastics needed.
